### PR TITLE
Add note for phase 1 of 20 docs in 20 days

### PR DESCRIPTION
**Pull request by kirbyk**
Hey @jdestefano-mongo the Atlas SWAT team wants to place some calls to actions (CTAs) in some highly trafficked / Atlas relevant sections of the Docs. The homepage of the BI Connector Docs is one of those pages. Over the next few weeks we'll see how this CTA performs and will experiment with variants accordingly.

Can you review these changes?

Changes staged at the following page:
https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/kirby/kk-20-docs-in-20-days-phase-1/index.html

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -53,12 +53,15 @@ describe possible configurations for a BI system.
...
-hosted solution for both MongoDB and |bi-short|. Atlas dedicated
-clusters (`size M10+
-<https://docs.atlas.mongodb.com/create-new-cluster/>`_)
-offer an `integrated BI Connector instance
-<https://docs.atlas.mongodb.com/bi-connection>`_ as an option.
+.. note::
+
+   If you don’t want to manage and administer the BI Connector, you can use
+   MongoDB Atlas, our fully-managed database-as-a-service. Databases hosted on
+   Atlas natively support the MongoDB Connector for BI and make it easy to
```
Comment (jdestefano-mongo) [inline source/index.txt]: s / MongoDB Connector for BI / |bi|

File: source/index.txt [hunk 2]
```diff
@@ -53,12 +53,15 @@ describe possible configurations for a BI system.
...
+   start analyzing your data with your preferred BI tools. `Create an Atlas
```
Comment (jdestefano-mongo) [inline source/index.txt]: Can we segue this last sentence in a little better? Suggest something like:

- For more information, see Create an Atlas Cluster.
- To get started, Create an Atlas Cluster
- Create an Atlas Cluster to get started.

etc.

File: source/index.txt [hunk 3]
```diff
@@ -53,12 +53,15 @@ describe possible configurations for a BI system.
...
 
-`MongoDB Atlas <https://www.mongodb.com/cloud/atlas>`_ provides a fully
-hosted solution for both MongoDB and |bi-short|. Atlas dedicated
-clusters (`size M10+
-<https://docs.atlas.mongodb.com/create-new-cluster/>`_)
-offer an `integrated BI Connector instance
-<https://docs.atlas.mongodb.com/bi-connection>`_ as an option.
+.. note::
+
+   If you don’t want to manage and administer the BI Connector, you can use
```
Comment (jdestefano-mongo) [inline source/index.txt]: Manage and administer are kind of synonymous.  What if we pick one and highlight the hassle of BIC installation...

"If you don't want to install and [manage | administer] the |bi-short|, you can use..."

Comment (jdestefano-mongo) [reply source/index.txt]: Regardless of the comment above, please:

s / "BI Connector" / |bi-short|

Comment (jdestefano-mongo) [reply source/index.txt]: NIT: "If you don't want to ..."  feels so marketing-y, like we're accusing readers of being lazy.

I'm going to follow that nit up with a suggestion that I don't think is that great, what are your thoughts on:

"The |bi| is now [hosted | available] in MongoDB Atlas, our fully-managed database-as-a-service, where you don't have to install or manage your own |bi-short| instance. <rest of your blurb>"

File: source/index.txt (hunk 1 above)
Comment (kirbyk) [reply source/index.txt]: Oh neat. Good to know! Where can I see a list of all of those?

File: source/index.txt (hunk 2 above)
Comment (kirbyk) [reply source/index.txt]: I like the last the best. Will do that.

File: source/index.txt (hunk 3 above)
Comment (kirbyk) [reply source/index.txt]: This last comment is a good suggestion. If it's okay, I'd like to keep the text as is (minus your other suggestions) and try suggestion in a future variant / experiment in a future week.

**Comment by kirbyk** [comment]
Thanks for all the feedback @jdestefano-mongo! Ready for another pass!

File: source/index.txt (hunk 1 above)
Comment (jdestefano-mongo) [reply source/index.txt]: `conf.py`

File: source/index.txt (hunk 3 above)
Comment (jdestefano-mongo) [reply source/index.txt]: Sure thing

**Pull request merged**